File: src/main/java/com/elf/vipForElf/web/dto/NewContractDTO.java
package com.elf.vipForElf.web.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class NewContractDTO {
    Long realEstateListingId;
    Long realEstateAgencyId;
    String landLordId;
    OffsetDateTime moveInDt;
}

File: src/main/java/com/elf/vipForElf/web/dto/NewContractDTO.kt
package com.elf.vipForElf.web.dto

import java.time.OffsetDateTime

data class NewContractDTO(
    val realEstateListingId: Long?,
    val realEstateAgencyId: Long?,
    val landLordId: String?,
    val moveInDt: OffsetDateTime?
)
